packages/reframed/src/reframed.ts

```diff
@@ -207,6 +242,12 @@ function monkeyPatchIFrameDocument(
 			},
 		},
 
+		readyState: {
+			get() {
+				return shadowRoot.reframeMetadata.iframeDocumentReadyState;
```

Comment:
I've still freaking out a bit about #36 as it will really hurt us when some of the resources in the fragments HTML stream take too long to load.

This can only happen when `iframeDocumentReadyState === 'completed'`.

So what if we made it easier for ourselves to know when we hit this case by checking here for this state and issuing a warning? something like:

```
if (shadowRoot.reframeMetadata.iframeDocumentReadyState === 'completed') {
  console.warn('reframed warning: `document.readyState` possibly returned `'completed'` prematurely. If your app is not working correctly, please see https://github.com/web-fragments/web-fragments/issues/36  and comment on this issue so that we can prioritize fixing it.');
}
```

Reply:
Yes, I can see your point 👍 

Personally I think that hitting this might be a bit edge casey, but I also appreciate that if someone were to encounter this it could be very very tough to debug and understand what's going on

I don't mind adding the warning, hopefully we'll remove it soon enough anyways 🙂👍